### PR TITLE
delly 0.7.3

### PR DESCRIPTION
**Pull request by ilovezfs**
- update to new version
- grab the tests from the 0.6.5 tar file since upstream removed them probably due to their size (about 27 MB).

### CODE REVIEW

**Comment by ilovezfs** [comment]
On Mavericks the test currently fails for 0.6.5 with the following:
```
vm01:~ openzfs$ brew test delly
Testing homebrew/science/delly
==> Using the sandbox
==> delly --outfile=/private/tmp/delly20160329-78582-1mpdo5d/test.vcf /usr/local/Cellar/delly/0.6.5/s
Last 15 lines from /Users/openzfs/Library/Logs/Homebrew/delly/01.delly:
2016-03-29 16:46:07 +0900

delly
--outfile=/private/tmp/delly20160329-78582-1mpdo5d/test.vcf
/usr/local/Cellar/delly/0.6.5/share/delly/test/DEL.bam

Error: homebrew/science/delly: failed
Failed executing: delly --outfile=/private/tmp/delly20160329-78582-1mpdo5d/test.vcf /usr/local/Cellar/delly/0.6.5/share/delly/test/DEL.bam
/usr/local/Library/Homebrew/formula.rb:1481:in `block in system'
/usr/local/Library/Homebrew/formula.rb:1418:in `open'
/usr/local/Library/Homebrew/formula.rb:1418:in `system'
/usr/local/Library/Taps/homebrew/homebrew-science/delly.rb:50:in `block in <class:Delly>'
/usr/local/Library/Homebrew/formula.rb:1327:in `block in run_test'
/usr/local/Library/Homebrew/extend/fileutils.rb:37:in `mktemp'
/usr/local/Library/Homebrew/formula.rb:1323:in `run_test'
/usr/local/Library/Homebrew/test.rb:28:in `block in <main>'
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/timeout.rb:66:in `timeout'
/usr/local/Library/Homebrew/test.rb:27:in `<main>'
vm01:~ openzfs$
```

A revision bump would fix this, too, but we might as well update the formula while we're at it.

File: delly.rb [hunk 1]
```diff
@@ -21,18 +21,26 @@ class Delly < Formula
...
 
-  resource "linux-binary-0.6.5" do
-    url "https://github.com/tobiasrausch/delly/releases/download/v0.6.5/delly_v0.6.5_CentOS5.4_x86_64bit"
-    sha256 "56ccdbd5e21d3570c7e34c82593e9ef5d86509f23f57e3e364bcd3f5a7e6899c"
+  resource "linux-binary" do
+    url "https://github.com/tobiasrausch/delly/releases/download/v0.7.2/delly_v0.7.2_CentOS5.4_x86_64bit"
+    sha256 "fd01ee996dbac497b0d151634c2c385f6c3ef78d64aa5daae25823ce6bf59eb8"
+  end
+
+  # The tests were removed after 0.6.5
```
Comment (ilovezfs) [inline delly.rb]: Does it make sense to grab the tests from the "old" tar file or should just a simple version test be used instead?

iMac-TMP:Homebrew joe$ du -sh delly-0.6.5.tar.gz 
 29M	delly-0.6.5.tar.gz
iMac-TMP:Homebrew joe$ du -sh delly-0.7.2.tar.gz 
2.4M	delly-0.7.2.tar.gz

I believe is the only reason upstream dumped them.

**Comment by ilovezfs** [comment]
@sjackman @tseemann mind reviewing this? (note: I updated it from 0.7.2 to 0.7.3 since there was a release subsequent to my opening the PR.)

**Comment by sjackman** [comment]
Looks good to me. (though I haven't tested it out). Thanks for your meticulous work, Joe.
Do you use these tools yourself, `delly` and `reapr`?

**Comment by ilovezfs** [comment]
@sjackman you're welcome. No, this is actually my first encounter with them.

**Pull request closed without merging**